plugins/slo/pkg/slo/slo_test.go

```diff
@@ -586,13 +606,14 @@ var _ = Describe("Converting SLO information to Cortex rules", Ordered, Label(te
 		Specify("The individual parts of the raw SLI queries should return data from cortex", func() {
```

Comment:
there is a todo comment a few lines above here that we should address

Reply:
prefer addressing it as a separate PR